### PR TITLE
log command query times

### PR DESCRIPTION
**Pull request by fdietze**

### CODE REVIEW

File: odor/src/main/scala/PostgresClient.scala [hunk 1]
```diff
@@ -109,14 +110,24 @@ class PostgresClient(val pool: PostgresConnectionPool)(implicit ec: ExecutionCon
...
+    val startTimeNanos = nowNano()
+    val result = await(
+      conn
         .query(
           command.sql,
           command.encoder.encode(params).map(_.orNull).toJSArray,
         )
         .toFuture,
     )
+    val affectedRows = Try(result.rowCount.toInt).orElse(Try(result.rows.length)).getOrElse(0)
```
Comment (cornerman) [inline odor/src/main/scala/PostgresClient.scala:123]: Can move this line into the `if` :)

Why is this not sufficient?
```
    val affectedRows = result.rowCount.toIntOption.getOrElse(result.rows.length)
```

Comment (fdietze) [reply odor/src/main/scala/PostgresClient.scala:123]: toIntOption does not exist on Double. And it's still crashing even with `Try`. I don't know why...

Comment (fdietze) [reply odor/src/main/scala/PostgresClient.scala]: found something that seems to work.

Comment (cornerman) [reply odor/src/main/scala/PostgresClient.scala]: You are right, it just takes the int part of the double.

Comment (cornerman) [reply odor/src/main/scala/PostgresClient.scala]: But how could `toInt` fail on a double? Is it null?

Comment (fdietze) [reply odor/src/main/scala/PostgresClient.scala]: > But how could `toInt` fail on a double?

javascript

**Pull request merged**